components/MoveInfo.js

```diff
@@ -0,0 +1,91 @@
+import styled from "styled-components";
+
+export default function MoveInfo({ previousMove, moveStatus }) {
+  console.log(previousMove);
+  console.log(moveStatus);
+
+  let piece;
+  switch (previousMove.piece) {
+    case "p":
+      piece = "Pawn";
+      break;
+    case "b":
+      piece = "Bishop";
+      break;
+    case "n":
+      piece = "Knight";
+      break;
+    case "r":
+      piece = "Rook";
+      break;
+    case "q":
+      piece = "Queen";
+      break;
+    case "k":
+      piece = "King";
+      break;
+    default:
+      piece = "piece";
```

Comment:
I love the use of Switch Statements. You could refactor it tho instead of using let and break. Try wrapping it into a function and return your desired piece strings. Then you can do something like:
```js
const piece = resolvePiece(previousMove.piece);

function resolvePiece(piece){
   switch (piece) {
      case "p":
          return "Pawn";
      case "b":
          return "Bishop";
      case "n":
          return "Knight";
      case "r":
          return "Rook";
      case "q":
          return "Queen";
      case "k":
          return "King";
      default:
          return "piece";
}
```